packages/internal/src/ts2js.ts

```diff
@@ -48,25 +50,33 @@ export const convertTsFilesToJs = (cwd: string, files: string[]) => {
       fs.unlinkSync(path.join(cwd, f))
     }
   }
+}
 
-  if (fs.existsSync(path.join(cwd, 'api/tsconfig.json'))) {
-    fs.renameSync(
-      path.join(cwd, 'api/tsconfig.json'),
-      path.join(cwd, 'api/jsconfig.json')
-    )
-  }
-  if (fs.existsSync(path.join(cwd, 'web/tsconfig.json'))) {
-    fs.renameSync(
-      path.join(cwd, 'web/tsconfig.json'),
-      path.join(cwd, 'web/jsconfig.json')
-    )
-  }
-  if (fs.existsSync(path.join(cwd, 'scripts/tsconfig.json'))) {
-    fs.renameSync(
-      path.join(cwd, 'scripts/tsconfig.json'),
-      path.join(cwd, 'scripts/jsconfig.json')
-    )
-  }
+/**
+ * @param {string} cwd
+ */
+export function convertTsConfigsToJsConfigs(cwd = getPaths().base) {
+  const tsConfigs = [
+    'api/tsconfig.json',
+    'web/tsconfig.json',
+    'scripts/tsconfig.json',
+  ]
+
+  tsConfigs
+    .filter((tsConfig) => fs.existsSync(path.join(cwd, tsConfig)))
+    .map((tsConfig) => readJSONSync(path.join(cwd, tsConfig)))
+    .map(convertTsConfigToJsConfig)
+    .forEach((jsConfig, i) => {
+      writeJSONSync(
```

Comment:
Not a big deal but the jsconfig json produced isn't formatted nicely. I wonder if there is the equivalent of `JSON.stringify({ a:1, b:2, c:3 }, null, 4)` to format it a little better with writeJsonSync?

Reply:
Ah missed that—it looks like the [`spaces` option](https://github.com/jprichardson/node-fs-extra/blob/master/docs/writeJson-sync.md#writejsonsyncfile-object-options) does what we want.

Done in https://github.com/redwoodjs/redwood/pull/4263/commits/9a19d8f4556efe87c5a797072252739de234d6b6 and https://github.com/redwoodjs/redwood/pull/4263/commits/3b23d713daca452d79e6b7f2a832d08f27dc62e9 (left a console log).